ui/src/app/components/rename-modal.tsx

```diff
@@ -26,44 +26,6 @@ const styles = reactStyles({
   },
 });
 
-/* Name validation for resources.
```

Comment:
moved to utils/resources, because it was used in contexts besides rename-modal - even before this PR.